app/js/components/LabResultEntry.jsx

```diff
@@ -166,6 +166,25 @@ export class LabResultEntry extends PureComponent {
               </div>
             )
         }
+        <div className="estimated-checkbox">
+          <Obs
+            conceptAnswer="87f506e3-4433-40ec-b16c-b3c65e402989"
+            widget="checkbox"
+            concept="87f506e3-4433-40ec-b16c-b3c65e402989"
+            path="estimated-checkbox"
+            checkBoxTitle="estimated"
+          />
+        </div>
+        <div className="test-location">
+          <span className="test-location-label">Reason:&nbsp;</span>
+          <Obs
```

Comment:
This test location, is only here for visuals, I'll wait for the final word on (https://tickets.pih-emr.org/secure/RapidBoard.jspa?rapidView=68&view=detail&selectedIssue=UHM-3518) and update it.